app/controllers/curation_concerns/scanned_resources_controller.rb

```diff
@@ -48,6 +48,17 @@ def bulk_edit
     @members = presenter.file_presenters
   end
 
+  def structure
+    @members = presenter.file_presenters
+    @logical_order = presenter.logical_order_object
+  end
+
+  def save_structure
+    curation_concern.logical_order.order = { "nodes": params["nodes"] }
+    curation_concern.save
+    head 200
+  end
+
```

Comment:
I like the naming of the methods -- I wonder if we should rename bulk_edit/save_order to match?